modules/openthread/platform/diag.c

```diff
@@ -92,18 +92,25 @@ otError otPlatDiagProcess(otInstance *aInstance, uint8_t aArgsLength, char *aArg
 	return OT_ERROR_NOT_IMPLEMENTED;
 }
 
-void otPlatDiagModeSet(bool aMode)
+static void log_error_returned(otError aError, const char * aName)
```

Comment:
Please use snake case for Zephyr code (i.e. functions not implementing OT API), so `error`, `name`